app/controllers/invite_requests_controller.rb

```diff
@@ -2,8 +2,10 @@ class InviteRequestsController < ApplicationController
   before_action :admin_only, only: [:manage, :destroy]
 
   # GET /invite_requests
+  # Set Browser Page Title to Invitation Requests | Archive Of Our Own
   def index
     @invite_request = InviteRequest.new
+    @page_title = t(".browser_page_title")
```

Comment:
If we use `@page_subtitle` instead of `@page_title`, that will automatically include ` | Archive of Our Own` after the subtitle (more dynamically, since that will use the configured archive name). Reference in the application template: https://github.com/otwcode/otwarchive/blob/4e35e9eebbe0fa8d1beb9a2d4c141efbfcd59592/app/views/layouts/application.html.erb#L23